pkg/distributor/distributor.go

```diff
@@ -109,7 +109,6 @@ type Config struct {
 	BillingConfig billing.Config             `yaml:"billing,omitempty"`
 	PoolConfig    ingester_client.PoolConfig `yaml:"pool,omitempty"`
 
-	EnableHATracker bool            `yaml:"enable_ha_tracker,omitempty"`
```

Comment:
Not sure you can remove this completely - the distributor won't start if you don't provide consul/etc address for the ha tracker.

The idea was to move this flag into the HA tracker config instead, and have that struct 'know' if it were enabled or not - and not start the consul client, loop etc if it is.

Reply:
:man_facepalming: